PythonServer.py

```diff
@@ -1,16 +1,54 @@
+import json
+from pathlib import Path
+from fastai import *
+from fastai.vision import *
 import zerorpc
 
-from test_python import test_me
+# SETUP HERE
+YOUR_CLASSES_HERE = ['black', 'grizzly', 'teddys'] # Your class labels
```

Comment:
@zcaceres : I think 1 change we should make is moving all the fastai specific stuff to a separate file (say `model_fastai.py`)

Idea of PythonServer.py was to keep it as an interface - like a relay-er (capable calling/passing around many different actions like train/test/predict/tokenize/etc)

that way we can also add support for say `model_keras.py` etc